package.json

```diff
@@ -35,7 +34,6 @@
     "react-spring": "^8.0.25",
```

Comment:
If I'm not mistaken, most of these dependencies like `react-dom`, `react`, `codemirror` are bundled by Webpack, but since this package is published for the exchange, we probably want to move these into `devDependencies` as well or make a separate folder for the extension itself.

@andyrichardson was also talking about making a separate repository for this, but in the short-term we'll probably want to publish a quick patch for the `@urql/devtools` exchange package without these dependencies :)

Reply:
So actually every dependency except `wonka` should move to devDependencies?

Reply:
Probably! 😅

Reply:
Will get on that

Reply:
on this note @andyrichardson, shall we open an issue to rewrite the introspection w/o using Apollo's `apollo-link-http` 😅